source/Quantities.Serialization/Text.Json.Test/FrequencySupportTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Quantities.Serialization.Text.Json.Test;
+
+public class FrequencySupportTest : IInjectedUnitTester<Frequency>
+{
+
+    [Theory]
+    [MemberData(nameof(InjectingAreas))]
+    public void DeserializationSupportsInjectedUnits(Frequency quantity)
+    {
+        const Double scalar = 2;
+        Time expectedTime = scalar / quantity;
+        Frequency deserializedFrequency = quantity.SupportsSerialization();
```

Comment:
I forgot what the convenience method `SupportsSerialization` did. All caught up now :)

Reply:
Since this does also an Assert, maybe move it to "act" section of the test.

Reply:
Fixed